server/circle/client.go

```diff
@@ -0,0 +1,216 @@
+}
+
+// GetRecentlyBuiltPipelines get all recently built pipelines in a org
+func GetRecentlyBuiltPipelines(apiToken string, orgSlug string, mine bool) ([]circleci.Pipeline1, error) {
+	pl, resp, err := client.PipelineApi.ListPipelines(getContext(apiToken), orgSlug, mine, nil)
+	resp.Body.Close()
+	if err != nil {
+		return nil, err
+	}
+
+	return pl.Items, nil
+}
+
+// GetAllPipelinesForProject get all pipelines for a given project
+func GetAllPipelinesForProject(apiToken string, projectSlug string) ([]circleci.Pipeline1, error) {
+	pl, resp, err := client.PipelineApi.ListPipelinesForProject(getContext(apiToken), projectSlug, nil)
+	resp.Body.Close()
+	if err != nil {
+		return nil, err
+	}
+
+	return pl.Items, nil
+}
+
+// GetAllMyPipelinesForProject get all pipelines triggered by you
+func GetAllMyPipelinesForProject(apiToken string, projectSlug string) ([]circleci.Pipeline1, error) {
+	pl, resp, err := client.PipelineApi.ListMyPipelines(getContext(apiToken), projectSlug, nil)
+	resp.Body.Close()
+	if err != nil {
+		return nil, err
+	}
+
+	return pl.Items, nil
+}
+
+// GetWorkflowsByPipeline get all workflows by pipeline ID
+func GetWorkflowsByPipeline(apiToken string, pipelineID string) ([]circleci.Workflow1, error) {
+	wf, resp, err := client.PipelineApi.ListWorkflowsByPipelineId(getContext(apiToken), pipelineID, nil)
+	resp.Body.Close()
+	if err != nil {
+		return nil, err
+	}
+
+	return wf.Items, nil
+}
+
+// GetNameByID returns username from user id
+func GetNameByID(apiToken string, id string) (string, error) {
+	user, resp, err := client.UserApi.GetUser(getContext(apiToken), id)
+	resp.Body.Close()
+	return user.Name, err
+}
+
+// TriggerPipeline triggers pipeline for given project and given branch/tag
+func TriggerPipeline(apiToken string, projectSlug string,
+	params circleci.TriggerPipelineParameters) (circleci.PipelineCreation, error) {
+	var opts = circleci.PipelineApiTriggerPipelineOpts{}
+	opts.Body = optional.NewInterface(params)
+
+	pl, resp, err := client.PipelineApi.TriggerPipeline(getContext(apiToken), projectSlug, &opts)
+	resp.Body.Close()
+	return pl, err
+}
+
+// GetPipelineByNum get info about single pipeline
+func GetPipelineByNum(apiToken string, projectSlug string, num string) (circleci.Pipeline, error) {
+	pl, resp, err := client.PipelineApi.GetPipelineByNumber(getContext(apiToken), projectSlug, num)
+	resp.Body.Close()
+	return pl, err
+}
+
+// GetPipelineByID get info about single pipeline
+func GetPipelineByID(apiToken string, pipelineID string) (circleci.Pipeline, error) {
+	pl, resp, err := client.PipelineApi.GetPipelineById(getContext(apiToken), pipelineID)
+	resp.Body.Close()
+	return pl, err
+}
+
+// RerunWorkflow reruns a given workflow
+func RerunWorkflow(apiToken string, workflowID string) (circleci.MessageResponse, error) {
+	ms, resp, err := client.WorkflowApi.RerunWorkflow(getContext(apiToken), workflowID, nil)
+	resp.Body.Close()
+	return ms, err
+}
+
+// CancelWorkflow reruns a given workflow
```

Comment:
nit:  verb doesn't match the function name.
```suggestion
// CancelWorkflow cancels a given workflow
```

Reply:
Done :white_check_mark: